### PR TITLE
Deprecate autostep in RHEL8

### PR DESCRIPTION
**Pull request by VladimirSlavik**
It doesn't really work anymore.

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
Only as draft because it's not clear yet if we will do this.

File: tests/commands/autostep.py
Comment (VladimirSlavik) [inline tests/commands/autostep.py:22]: No need to import `RHEL8_AutoStep`

File: pykickstart/commands/autostep.py
Comment (VladimirSlavik) [inline pykickstart/commands/autostep.py:20]: Import `versionToLongString` here.

**Review by VladimirSlavik — COMMENTED** [review]
After landing the F34 changes in #344, some errors in this are apparent.

**Comment by VladimirSlavik** [comment]
It seems to me that Pylint got upgraded in Travis (with new Ubuntu that they warned of), the errors don't look to me like "mine".

**Comment by VladimirSlavik** [comment]
@bcl this is blocked by #371, right?

**Comment by bcl** [comment]
Not really, I can run the tests locally before merging. But it does need a rhbz with acks.

**Review by bcl — CHANGES_REQUESTED** [review]
Rebase this PR on the current rhel8-branch so that the new GH Action based tests run.

The Commit message also needs a Resolves: rhbz#1895951 line added to it.

**Comment by VladimirSlavik** [comment]
Thanks!

**Pull request merged**
